File: sdks/typescript/examples/account/get_balance.ts
import { config } from 'dotenv'; config({ path: '../../.env' });
import pmxt from 'pmxtjs';

(async () => {
    const client = new pmxt.Polymarket({
        privateKey: process.env.POLYMARKET_PRIVATE_KEY,
        proxyAddress: process.env.POLYMARKET_PROXY_ADDRESS,
        signatureType: 'gnosis-safe'
    });
    console.log(await client.fetchBalance());
})();

File: sdks/python/examples/account/get_balance.py
import os
import pmxt

def main():
    client = pmxt.Polymarket(
        private_key=os.getenv("POLYMARKET_PRIVATE_KEY"),
        proxy_address=os.getenv("POLYMARKET_PROXY_ADDRESS"),
        signature_type='gnosis-safe'
    )
    print(client.fetch_balance())

if __name__ == "__main__":
    main()
